City-of-Helsinki/helsinki-paatokset
# PP-95: Add default configuration, enable sub-theme, update Drupal core.

**To test:**
* Check out branch
* Run `make new`
  * This might output a few errors, but should complete successfully
* The script should install a new site based on existing config instead of the default profile:
  * Site name should now be "Päätökset"
  * The hel_paatokset sub theme should now be enabled
  * Everything should be up-to-date: https://helsinki-paatokset.docker.sh/fi/admin/reports/updates (or at least Drupal Core should be 9.1.7)
